install.sh

```diff
@@ -0,0 +1,7 @@
+git clone https://github.com/Stock-Engine/stock-engine-frontend.git
+
+cd /stock-engine-frontend
+
+npm install
```

Comment:
User may not have npm installed, please  rebase to the other PR after it's merged and add node installation the same way as pipenv